src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -54,6 +56,12 @@ import {
     shouldShowBrokenConnectionViolationForMultipleTransactions,
 } from './TransactionUtils';
 
+let allBetas: OnyxEntry<Beta[]>;
+Onyx.connect({
+    key: ONYXKEYS.BETAS,
+    callback: (value) => (allBetas = value),
+});
+
```

Comment:
Let's not add any other calls to onyx.connect, we can call useOnyx here like we did in other places:
https://github.com/Expensify/App/blob/baab827fd3065c42b37ece0c891785d053baabde/src/components/Search/SearchAutocompleteList.tsx#L153